### PR TITLE
Pill bottles show how many pills they have in overlays

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Title, just lick item stacks

## Changelog
:cl:
qol: Pill bottles show how many pills they have in overlays
/:cl:

### CODE REVIEW

**Review by KrabSpider — COMMENTED** [review]
These weren't really mentioned in the PR body...
I take it this was a slip up?

File: code/modules/mob/living/carbon/xenomorph/castes/beetle/castedatum_beetle.dm [hunk 1]
```diff
@@ -21,7 +21,7 @@
 	plasma_gain = 10
 
 	// *** Health *** //
-	max_health = 400
+	max_health = 260
```
Comment (KrabSpider) [inline code/modules/mob/living/carbon/xenomorph/castes/beetle/castedatum_beetle.dm:24]: Uhhh?

File: code/modules/ai/ai_behaviors/ai_behavior.dm [hunk 2]
```diff
@@ -35,7 +35,7 @@ Registers signals, handles the pathfinding element addition/removal alongside ma
 	///When this timer is up, we force a change of node to ensure that the ai will never stay stuck trying to go to a specific node
 	var/anti_stuck_timer
 	///Minimum health percentage before the ai tries to run away
-	var/minimum_health = 0.4
+	var/minimum_health = 0
```
Comment (BraveMole) [inline code/modules/ai/ai_behaviors/ai_behavior.dm:38]:
```suggestion
	var/minimum_health = 0.4
```

**Comment by novaepee** [comment]
speedmerge now

**Pull request merged**